x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/LimitOperator.java

```diff
@@ -38,25 +42,32 @@ public class LimitOperator implements Operator {
      */
     private long rowsEmitted;
 
-    private Page lastInput;
-
     private final Limiter limiter;
     private boolean finished;
 
-    public LimitOperator(Limiter limiter) {
+    private final int pageSize;
+    private int pendingRows;
```

Comment:
Could you move this one below the `final` ones? I just want to keep the mutable ones not mixed in.

Reply:
Sure, I regrouped these in https://github.com/elastic/elasticsearch/pull/128531/commits/c94c72ca9a3075e6a39678692d65a0a0bf108ab5

Reply:
Thanks!